test/test_urdf_parser.jl

```diff
@@ -0,0 +1,32 @@
+@testset "URDF parser" begin
+    @testset "joint bounds" begin
+        acrobot = parse_urdf(Float64, joinpath(@__DIR__, "urdf", "Acrobot.urdf"))
+        @test findjoint(acrobot, "shoulder").bounds.position.lower == [-Inf]
+        @test findjoint(acrobot, "shoulder").bounds.position.upper == [Inf]
+        @test findjoint(acrobot, "shoulder").bounds.velocity.lower == [-Inf]
+        @test findjoint(acrobot, "shoulder").bounds.velocity.upper == [Inf]
+        @test findjoint(acrobot, "shoulder").bounds.effort.lower == [-Inf]
+        @test findjoint(acrobot, "shoulder").bounds.effort.upper == [Inf]
```

Comment:
Add accessor functions:
* `position_bounds(::Joint)`
* `velocity_bounds(::Joint)`
* `effort_bounds(::Joint)`

and use here. Accessors for `Bounds` variables are not necessary.

Reply:
Added. Is it OK to keep the `Bounds` accessors? I'm using them in my simulation code.